sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala

```diff
@@ -499,6 +489,8 @@ abstract class BinaryOperator extends BinaryExpression with ExpectsInputTypes {
 
   def symbol: String
 
+  def sqlOperator: String = symbol
```

Comment:
For example, `And.symbol` is `&&`, while `And.sqlOperator` is `AND`. The former is used in Hive shims for converting Hive filters. The latter is used to generate SQL string.